website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,33 @@
+## Usage
+
+### When to use
+
+- Use a file input to let users quickly select files from their computer.
+
+### Disabled State 
+
+The File Input component lets users add documents and images. Disabling it can make users feel confused, frustrated, and unsure about why it's not working.
+
+Instead of disabling, consider these alternatives:
+
+- Include explanatory text or prompts: If you display a disabled file input, explain why it's disabled and what users need to do to enable it. You can use a tooltip, or an instructional sentence.
+
+- Use progressive disclosure: Display or enable the file input only when users meet certain conditions (e.g., after completing the necessary information). This approach guides users step-by-step and presents relevant options based on their progress.
+
+### Truncation
+
+Truncation works differently in Figma and the browser. In Figma, truncation occurs at the end of the text, while in the browser, it will truncate text in the middle.
+
+![visual difference in truncation between figma and browser](/assets/components/form/file-input/file-input-truncation.png)
+
+### Multiple Files 
+
+In the native HTML file input, you can input multiple files using the "multiple" attribute. When you input more than one file, the status message will change to show the number of files you've selected, such as "2 files" in Chrome (Other browsers may have slight differences in how they display this).
+
+### Browser differences
+
+The appearance of file input can vary across different web browsers.
+
+File Input accepts [all native HTML types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file), but we offer built in styling for the following:
```

Comment:
@andgen404  It actually doesn't. The input `type` is set specifically to "file".

HTML: `<input type="file" />`

Reply:
```suggestion
File Input accepts [HTML: <input type="file" />](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file), but we offer built in styling for the following:
```

Reply:
So would I write: File Input accepts `HTML: <input type="file" />`, and we offer built in styling for the following:

Reply:
No that's just what the actual code is under the hood. I can update this line for you if you like.

Reply:
Ah, okay. Thank you